sdv/lite/tabular.py

```diff
@@ -49,15 +49,16 @@ def __init__(self, name=None, metadata=None, constraints=None):
                           'detected from your data. This process may not be accurate. '
                           'We recommend writing metadata to ensure correct data handling.')
 
-        if metadata is not None and constraints is not None:
+        else:
```

Comment:
Wouldn't this now fail if `metadata` is `None` and `constraints` are not `None`?